tests/tests-environment.sh

```diff
@@ -17,14 +17,13 @@ sudo dpkg -i patchelf_0.8-2_amd64.deb
 # make -j$(nproc)
 # sudo make install
 
-cd /tmp/
+pushd /tmp/
```

Comment:
Is there any advantage of `pushd` over `cd`? I like `cd` better because everyone understands it. Am I missing something?

Reply:
It handles recursion properly, `-` is unreliable. It's well understood by regular shell users, too.